From 33531a448003c7e167939c2f647afbb316ee7ba4 Mon Sep 17 00:00:00 2001
From: Andrey Konovalov <andrey.konovalov@linaro.org>
Date: Wed, 28 Sep 2016 11:55:03 -0700
Subject: [PATCH] Change egl and wayland-egl version numbers

According to the sources, the driver supports EGL_VERSION_1_4.
So the version must be 14.x.x follow the mesa versioning rules.
The current version value of 8.14 looks wrong, as it confuses
other packages configuration scripts. E.g. chromium refuses to build
with EGL version 8.14.

Signed-off-by: Andrey Konovalov <andrey.konovalov@linaro.org>
---
 targetfs/jacinto6evm/lib/pkgconfig/egl.pc         | 2 +-
 targetfs/jacinto6evm/lib/pkgconfig/wayland-egl.pc | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/targetfs/jacinto6evm/lib/pkgconfig/egl.pc b/targetfs/jacinto6evm/lib/pkgconfig/egl.pc
index 1f26003..8b8028d 100644
--- a/targetfs/jacinto6evm/lib/pkgconfig/egl.pc
+++ b/targetfs/jacinto6evm/lib/pkgconfig/egl.pc
@@ -5,6 +5,6 @@ includedir=${prefix}/include
 
 Name: EGL
 Description: PowerVR EGL library
-Version: 8.14
+Version: 14.0.8
 Libs: -L${libdir} -lEGL -lIMGegl -ldrm_omap -lsrv_um
 Cflags: -I${includedir}
diff --git a/targetfs/jacinto6evm/lib/pkgconfig/wayland-egl.pc b/targetfs/jacinto6evm/lib/pkgconfig/wayland-egl.pc
index 01163c6..529100a 100644
--- a/targetfs/jacinto6evm/lib/pkgconfig/wayland-egl.pc
+++ b/targetfs/jacinto6evm/lib/pkgconfig/wayland-egl.pc
@@ -5,6 +5,6 @@ includedir=${prefix}/include
 
 Name: pvr_wlegl
 Description: PowerVR pvr_wlegl library
-Version: 8.14
+Version: 14.0.8
 Libs: -L${libdir} -lpvr_wlegl 
 Cflags: -I${includedir}
-- 
2.1.4

